src/pages/dashboard/Dashboard.tsx

```diff
@@ -25,6 +26,7 @@ export function Dashboard() {
   return (
     <Default title={t('dashboard')}>
       <Breadcrumbs pages={navigation} />
+  <Totals></Totals>
```

Comment:
Minor, but we prefer short self-closing component tags if no children are present.